libcontainer/process_linux.go

```diff
@@ -141,6 +141,16 @@ func (p *setnsProcess) execSetns() error {
 		p.cmd.Wait()
 		return newSystemErrorWithCause(err, "reading pid from init pipe")
 	}
+
+	// Clean up the zombie parent process
+	firstChildProcess, err := os.FindProcess(pid.PidFirstChild)
+	if err != nil {
```

Comment:
Shouldn't firstChildProcess (runc:[1:CHILD]) always exist? (thus `wait` can't fail)?
If it doesn't exist we won't be able to get the grandchild's pid.